Documentation/Contributing/UnitTests.md

```diff
@@ -77,55 +76,107 @@ To create a new test the following template can be used:
+- Right click, Create > Testing > C# Test Script
+- Replace the default template with the example starter test below
+
+``` csharp
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System;
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    class ExamplePlayModeTests : BasePlayModeTests
+    {
+        // Override Setup() in BasePlayModeTests to change the starting camera position
+        public override void Setup()
         {
-            // call shutdown if you've created an mrtk GameObject in your test
-            TestUtilities.ShutdownMixedRealityToolkit();
+            base.Setup();
+            
+            // Set camera position to Vector3(0,0,0), the default position Vector3(1.0f, 1.5f. -2.0f)
+            TestUtilities.PlayspaceToOriginLookingForward();
         }
 
-
         #region Tests
 
+        /// <summary>
+        /// Example starter test that shows a test hand, gets the line pointer and moves the hand
+        /// </summary>
         [UnityTest]
-        /// the name of this method will be used as test name in the unity test runner
-        public IEnumerator TestMyFeature()
+        public IEnumerator ExampleTest()
         {
-            // write your test here
-            yield return null;
-        }
+            // Get the input system
+            var inputSystem = PlayModeTestUtilities.GetInputSystem();
+
+            // Create a new test hand for input
+            var rightHand = new TestHand(Handedness.Right);
+
+            // Initialize the position of the hand
+            Vector3 initialPos = new Vector3(0f, 0f, 0.5f);
 
+            // Show the hand
+            yield return rightHand.Show(initialPos);
+
+            // Get hand controller 
+            var handController = inputSystem.DetectedControllers.First(x => x.ControllerHandedness == Utilities.Handedness.Right && x.InputSource.SourceType == InputSourceType.Hand);
+            Debug.Assert(handController != null);
+
+            // Get the line pointer from the hand controller
+            var linePointer = handController.InputSource.Pointers.First(x => x is LinePointer);
+            Assert.IsNotNull(linePointer);
+
+            // Move the right hand to a new position
+            yield return rightHand.MoveTo(new Vector3(0, 0, 2.0f));
```

Comment:
@julenka 
Added in this comment:
```
            // We are doing a yield return here because moving the hand to a new position
            // requires multiple frames to complete the action.
            yield return rightHand.MoveTo(new Vector3(0, 0, 2.0f));
```